# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/hannes/hibernate/install-sh.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./install.sh.orig	2004-12-04 00:40:57.651911496 +0100
+++ ./install.sh	2004-12-04 00:41:12.443662808 +0100
@@ -14,18 +14,6 @@
 
 [ -z "$OLD_SCRIPTLET_DIR" ] && OLD_SCRIPTLET_DIR=$CONFIG_DIR/scriptlets.d
 
-# Test if the script is already installed.
-if [ -d $CONFIG_DIR -o -f $SCRIPT_DEST ] ; then
-    echo "Config directory $CONFIG_DIR and/or $SCRIPT_DEST already exist."
-    echo -n "Are you sure you want to overwrite them? (y/N) "
-    read REPLY
-    echo
-    case $REPLY in
-	y*|Y*) ;;
-	*) echo "Aborting!" ; exit 1 ;;
-    esac
-fi
-
 (
 set -e
 
